TM1py/Services/CellService.py

```diff
@@ -546,6 +546,27 @@ def execute_view_csv(self, cube_name: str, view_name: str, private: bool = False
                                         skip_rule_derived_cells=skip_rule_derived_cells, line_separator=line_separator,
                                         value_separator=value_separator, **kwargs)
 
+    def execute_mdx_elements_value_dict(self, mdx: str, top: int = None, skip: int = None, skip_zeros: bool = True,
+                                        skip_consolidated_cells: bool = False, skip_rule_derived_cells: bool = False,
+                                        value_separator: str = "|", **kwargs) -> Dict[str, Union[str, float]]:
+        """ Optimized for performance. Get Dict from MDX Query.
+
+        :param mdx: Valid MDX Query
+        :param top: Int, number of cells to return (counting from top)
+        :param skip: Int, number of cells to skip (counting from top)
+        :param skip_zeros: skip zeros in cellset (irrespective of zero suppression in MDX / view)
+        :param skip_consolidated_cells: skip consolidated cells in cellset
+        :param skip_rule_derived_cells: skip rule derived cells in cellset
+        :param value_separator: value separator the tuple
```

Comment:
perhaps it makes sense to call it `element_seperator` here

Reply:
done.